portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,39 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/*
+ * Adds a random quote to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomPrequelQuote() {
+  const quotes = [`Hello there.`, `There’s always a bigger fish.`, `I \
+    don’t like sand. It’s coarse and rough and irritating and it gets \
+	everywhere.`, `Now this is podracing!`, `I don’t care what universe \
+	you’re from, that’s got to hurt!`, `I sense Count Dooku.`, `His cells \
+	have the highest concentration of midi-chlorians I have seen in a \
+	life-form.`, `I AM the Senate!`, `I’m just a simple man, trying to \
+	make my way in the universe.`];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random quote.
+  const quote = quotes[Math.floor(Math.random() * quotes.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const quoteContainer = document.getElementById('quote-container');
+  quoteContainer.innerText = quote;
+}
+
+/*
+ * Changes profile picture to random image. 
+ */
+
+function changeProfilePic() {
+    const images = ["IMG_1502.png", "pfp_fr.jpg", "pfp_eg.jpg", "pfp_gr.jpg",
```

Comment:
Another small style guide thing is to always use single quotes for JavaScript: https://google.github.io/styleguide/jsguide.html#features-string-literals

Reply:
Fixed